public/styles.css

```diff
@@ -206,3 +206,13 @@ table > tbody > tr > td {
     border-radius: 8px;			
     pointer-events: none;			
 }
+.receivedGrid line {
+    stroke: lightgrey;
+    stroke-opacity: 0.7;
```

Comment:
make it more transparent so that the graphs don't look too busy

Reply:
Done